### ISSUE TITLE
Action Blocked-Sharing your account with a service that helps you get more likes

### ISSUE THREAD

**Issue by masto182** [InstaPy/InstaPy#4917]
Hot Sure if anyone else has had this, 

But I have been account blocked and I get the following message even on my phone.

Account Blocked

Your Account has been temporarily blocked from taking this action.
Sharing your account with a service that helps you get more likes of followers goes against our Community Guidelines.
This block will expire on 2019-08-22.  Tell us if you think we made a mistake.

I was using the dev branch, the follow_user_followers action.
I was running it headless=false, so that I could get passed the 2-factor auth.
It asked me to reset my password when I logged into my phone.
After updating it in both places I got the block.

**Comment by jonap339** [comment]
oh my God! This is serious :( Can you report the value of likes, follow-unfollow that you were using?

**Comment by spikeyhair** [comment]
The first time i get this block, it blocked me for 2 weeks. After the 2 weeks block, I waited for another 7 days before starting the bot again. This time I notice I have to go really slow, for example  max 4 or 5 likes/follow per hour only. Once I started to follow/like faster than that I get soft-block.

I take 24 hours rest after 1 or 2 soft-block before starting the bot again.

After trial and error, and get a few soft-block, I get a long-block again. Which I get the same message as masto182, will also expire on 2019-08-22.

My definition of soft-block means it showed "action blocked". once i clear cookies then can perform follow/like actions again.

So from my observation, I notice max like/follow/unfollow/comment is only 5 per hour. Going faster than that will trigger the soft-block, after a few soft-block, you will get a long-block. I am not sure will after a few long-block the whole account be removed or not. I don't dare to reach there.
